src/pages/data-visualization/design/basics.mdx

```diff
@@ -44,18 +44,8 @@ To make data visualization distinctive as IBM products, there are multiple eleme
 <Row>
 <Column colMd={5} colLg={8}>
 
-The title should reflect the main insight the data reveals. The legend should explain the chart’s meaning by defining the association of each visual property, such as color, shape and size, to the corresponding data. When possible, use labels directly on the chart to avoid long legends. All the text should be concise and easy to understand.
+The title should reflect the main insight the data reveals. The legend should explain the chart’s meaning by defining the association of each visual property, such as color, shape and size, to the corresponding data. When possible, use labels directly on the chart to avoid long legends. All the text should be concise and easy to understand. [Learn more on Carbon.](https://www.carbondesignsystem.com/data-visualization/legends)
```

Comment:
Should this be "Learn more **at** Carbon?" or "Learn more with the Carbon documentation" prepositions are hard haha.